manof/image.py

```diff
@@ -384,6 +387,10 @@ def privileged(self):
 
         return False
 
+    @property
+    def pid(self):
+        return ''
```

Comment:
```suggestion
        return None
```

Will be more consistent with other properties